fuzzers/baby/backtrace_baby_fuzzers/forkserver_executor/src/main.rs

```diff
@@ -25,7 +25,7 @@ use libafl_bolts::{
     rands::StdRand,
     shmem::{ShMem, ShMemProvider},
     tuples::tuple_list,
-    AsSliceMut,
+    AflTargetArgs, AsSliceMut,
```

Comment:
If every forkserver needs this now, maybe ForkserverTargetArgs? SubprocessTargetArgs?

Reply:
Just TargetArgs?